UnitystationLauncher/Services/CodeScanConfigService.cs

```diff
@@ -194,9 +194,9 @@ private string GetZipFolderName()
         }
     }
 
-    private string GetFolderName(string version)
+    public static string GetFolderName(string version, IEnvironmentService environmentService)
```

Comment:
is `envieronmentService`being injected now? I'm not familiar with this codebase. Where was it getting it from before?

Reply:
I think almost all pages use `envieronmentService` for one reason or another. I've just decided to make this function static and public, so I don't have to rewrite this logic elsewhere.